.travis.yml

```diff
@@ -53,6 +53,7 @@ jobs:
       os: linux
       arch: amd64
       install: "pip install nox"
+      if: env(DISCORD_EMOJI_MAPPING_URL) IS present
```

Comment:
This has been fixed as of #124, you can revert this

Reply:
no. it is secured, so it needs to be here.

securing the variable stops him being able to expose it.

Otherwise people could steal my passwords by just opening PRs with modifications that just send the encrypted variables to places.